client/src/modules/reports/reports.html

```diff
@@ -3,19 +3,23 @@
 
     <ol class="headercrumb">
       <li class="static" translate>TREE.REPORTS</li>
-      <li class="title" translate>{{ReportCtrl.report.title_key}}</li>
+      <li ng-hide="ReportCtrl.isArchive()" class="title" translate>{{ ReportCtrl.report.title_key }}</li>
+      <li ng-show="ReportCtrl.isArchive()">
+        <a data-method="report-config" ng-href="#!/reports/{{ReportCtrl.report.report_key}}" translate>{{ ReportCtrl.report.title_key }}</a>
```

Comment:
Can this use `ui-sref` instead of `ng-href`, by any chance?  It would be more in line with the rest of our modules.